packages/gestalt/src/Button.flowtest.js

```diff
@@ -3,24 +3,10 @@ import Button from './Button.js';
 
 const ValidDefaultButtonType = <Button text="Next" />;
 
-const ValidLinkRole = <Button text="Next" role="link" href="http://www.pinterest.com" />;
-
-const ValidSubmitType = <Button text="Next" type="submit" />;
-
 const NonExistingProp = (
-  // $FlowExpectedError[incompatible-type]
+  // $FlowExpectedError[prop-missing]
```

Comment:
interesting...

Reply:
Yes, I think this is a side effect of the disjoint union, the error now gets recognized as prop missing (which I think makes more sense in this context)